onnxruntime/test/onnx/main.cc

```diff
@@ -854,6 +854,9 @@ int main(int argc, char* argv[]) {
     fprintf(stderr, "%s\n", ex.what());
     retval = -1;
   }
+#ifdef _WIN32
+  Sleep(10); // Temporary workaround until we figure out what's crashing on exit due to a race condition
```

Comment:
Are we sure this will work during the day when load on the CI system is high?

Reply:
Interesting, that might explain why my tests succeed at night. I have a third version that might bypass this entirely after a suggestion from Changming...